hl7_2_fhir/huisartswaarneemgegevens/6.10.1.3/professionele_samenvatting/payload/QUPC_IN990002NL-2-BundleGeneralPractitioner.xsl

```diff
@@ -66,7 +66,11 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
     <xsl:template match="hl7:QUPC_IN990002NL">
         <xsl:choose>
             <xsl:when test="ancestor::hl7:MCCI_IN200101">
-                <xsl:message terminate="yes">Unsupported input MCCI_IN200101. Can only support singular QUPC_IN990002NL</xsl:message>
+                <xsl:call-template name="util:logMessage">
+                    <xsl:with-param name="level" select="$logWARN"/>
```

Comment:
This should be FATAL based on old version.

Reply:
Agree